web/crux/src/app/deploy/interceptors/deploy.create.interceptor.ts

```diff
@@ -26,7 +38,7 @@ export default class DeployCreateValidationInterceptor implements NestIntercepto
 
     if (existingDeployment) {
       throw new CruxConflictException({
-        message: 'There is already a deployment with preparing status for the version on that node',
+        message: version.type === VersionTypeEnum.rolling ? 'rollingVersionDeployment' : 'alreadyHavePreparing',
```

Comment:
Leave the message as an information for the developer.
Use a property and value instead. `property: 'version.type', value: `rolling`

Reply:
Unfortunatelly value must be the existingDeployment id because we need to redirect the user to the already existing deployment.

Reply:
As we discussed in person, you could add an `error?: string` field for this purposes.